pubnub-users/src/test/kotlin/com/pubnub/user/UserExtensionKtTest.kt

```diff
@@ -178,6 +180,22 @@ internal class UserExtensionKtTest {
         verify { pubNub.removeListener(any()) }
     }
 
+    @Test
+    internal fun can_modify_userId() {
+        pubNub.configuration.userId = UserId("modified")
```

Comment:
How do you know it's not `modified` already? Maybe use `UUID.randomUUID().toString()` instead?

Reply:
In setUp method there is 
`val pnConfiguration = PNConfiguration(PubNub.generateUUID())`
but for clarity I will modify this test to: 

```
    @Test
    internal fun can_modify_userId() {
        val modifiedUserId = "NewUserId-${UUID.randomUUID()}"
        pubNub.configuration.userId = UserId(modifiedUserId)

        assertEquals(modifiedUserId, pubNub.configuration.userId.value)
    }
```